applications/nrf5340_audio/tools/buildprog/nrf5340_audio_dk_devices.json

```diff
@@ -12,6 +12,6 @@
  {
         "nrf5340_audio_dk_snr": 1000,
         "nrf5340_audio_dk_dev": "headset",
-        "channel": "right"
+         "channel": "right"
```

Comment:
This file should be ignore

Reply:
`git update-index --assume-unchanged <file>`